### PR TITLE
Updated all images to use appropriate alt text and adjusted tiles to be clickable via keyboard navigation

### PR DESCRIPTION
**Pull request by jmkulwik**
This is part of a hackathon task to make the hourofcode.com/learn page more accessible and compliant with WCAG2A standards.

Due to this change, the lighthouse accessibility score from the chrome dev console improved from 65 to 85 out of 100 possible points.

This specifically focuses on the img elements on the page. 
Rule: "All non-text content that is presented to the user has a text alternative that serves the equivalent purpose..."
"Img element missing an alt attribute. Use the alt attribute to specify a short text alternative."
"Principle1.Guideline1_1.1_1_1.H37" 

The following images were updated:

alt=""
image

alt=""
image

Moved from an image to an icon:
Old:
image

New:
image

One exception to the rule above is when the image's alt text would be the same as its link text. In this case, the tile elements have images as well as description text. The whole element is used together as a button, and the image does not give semantic meaning beyond the tile's description text. Instead of adding alt text, I updated the tiles to be keyboard accessible.

tilesFocusable

## Links
[STAR-1201](https://codedotorg.atlassian.net/browse/STAR-1201)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/tutorialExplorer/roboticsButton.jsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ export default class RoboticsButton extends React.Component {
                 <img
                   src="/images/learn/robotics-link.png"
                   style={styles.roboticsButtonImage}
+                  alt="A humanoid robot"
```
Comment (breville) [inline apps/src/tutorialExplorer/roboticsButton.jsx]: This should come from the `i18n` string table.

File: pegasus/sites.v3/hourofcode.com/public/learn/index.haml [hunk 2]
```diff
@@ -134,6 +134,7 @@ social:
     var img = $('<img>');
     img.attr('src', studentImage);
     img.attr('style', 'opacity: 0');
+    img.attr('alt', 'Hour of Code events happening all around the globe');
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/learn/index.haml]: Have we confirmed the new strings with marketing/product?

**Review by molly-moen — APPROVED** [review]
This is awesome!

File: pegasus/sites.v3/hourofcode.com/public/learn/index.haml (hunk 2 above)
Comment (epeach) [reply pegasus/sites.v3/hourofcode.com/public/learn/index.haml]: Looking at this image in context, I wonder if this alt text could also be an empty string? This image is decorative and the associated text provides the necessary context to understand this section.

Comment (jmkulwik) [reply pegasus/sites.v3/hourofcode.com/public/learn/index.haml]: Good call. I'll do that.

File: apps/src/tutorialExplorer/roboticsButton.jsx (hunk 1 above)
Comment (jmkulwik) [reply apps/src/tutorialExplorer/roboticsButton.jsx]: Looking at this again in context, I think it makes more sense to set it to alt="". The image doesn't really have any semantic value.

**Pull request merged**